data/theleocodes.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Tanvi Saini",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "**MERN-Stack Developer | Cloud-Computing",
+    "tags": ["ReactJS", "JavaScript", "Cloud", "MongoDB", "NodeJS"],
+    "socials": [
+      { "icon": "FaTwitter", "url": "https://twitter.com/" },
```

Comment:
Do you want this to link to your twitter account? You need to add your twitter username at the end